docs/content/manual/_incoming/2274-vm-import.md

```diff
@@ -61,23 +110,46 @@ vcsim   clusterReady
 ```
 
 ### Verify OpenstackSource
-* Define a secret for your Openstack cluster
 
-```
-apiVersion: v1
-kind: Secret
-metadata: 
-  name: devstack-credentials
-  namespace: default
-stringData:
-  "username": "user"
-  "password": "password"
-  "project_name": "admin"
-  "domain_name": "default"
-  "ca_cert": "pem-encoded-ca-cert"
-```
-
-* Define an OpenstackSource Object
+1. Access the Openstack dashboard
+1. Create some images 
+![image](/docs/static/images/addons/2274-openstack-create-image.png)
+
+1. Create key pairs
+1. Create an empty volume to ensure the microstack.cinder services works well (if failed, check the OpenStack troubleshooting section)
+![image](/docs/static/images/addons/2274-openstack-empty-volume.png)
+![image](/docs/static/images/addons/2274-openstack-empty-volume-display.png)
+
+1. Launch a vm instance, ensure the vm can be created without error message
+![image](/docs/static/images/addons/2274-openstack-create-vm.png)
+
+1. Enable the vm-import-controller addon on Harvester
+![image](/docs/static/images/addons/2274-enable-vm-importer.png)
+
+1. Open Compute -> System -> System Information page, find the `Services` tab
+1. Check the `Region` in the page (eg. `microstack`)
+![image](/docs/static/images/addons/2274-openstack-region.png)
+1. Check the endpoint of the identity item
+![image](/docs/static/images/addons/2274-openstack-endpoint.png)
+
+1. Access the Harvester node machine
+1. Define a secret for your Openstack cluster in yaml file and use `kubectl` to create the crd
+
+    ```
+    apiVersion: v1
+    kind: Secret
+    metadata: 
+      name: devstack-credentials
+      namespace: default
+    stringData:
+      "username": "user" #dashboard username
+      "password": "password" #dashboard password
+      "project_name": "admin"
+      "domain_name": "default"
+      "ca_cert": "pem-encoded-ca-cert"
```

Comment:
Let's remove the `ca_cert`.
As juggling the certs for provisioning, OpenStack make things more difficult.

In the vm-import controller:
https://github.com/harvester/vm-import-controller/blob/148cec8766d0dfa3d363f0b85d6ec327430d677d/pkg/source/openstack/client.go#L76-L84

If we don't pass in a ca_cert, it will just turn on insecure_skip_verify.
Which MicroStack builds https, getting the cert out of MicroStack Snap and then neeeding to shovel it into Harvester is more complicated.
Additionally DevStack (MicroStack alternative) does the same thing.

Reply:
Thanks for the information and reminder. 
I just removed the `"ca_cert": "pem-encoded-ca-cert"` from the secret of devstack-credentials